cmd/api/main.go

```diff
@@ -24,27 +24,49 @@ import (
 )
 
 var (
-	dbConn *gorm.DB
-	conf   *configs.Configs
+	dbConn     *gorm.DB
+	grpcServer *grpc.Server
 )
 
 func init() {
 	log.SetFlags(log.LstdFlags | log.Lshortfile)
 
-	conf = configs.LoadFromEnv()
-
-	conn := &db.DBConn{
-		Info:       conf.DB,
-		SilentMode: conf.IsProduction(),
-	}
+	configs.LoadFromEnv()
 
 	var err error
-	if dbConn, err = conn.Open(); err != nil {
+	if dbConn, err = db.Open(); err != nil {
 		log.Fatal(err)
 	}
 }
 
 func main() {
+	/*
```

Comment:
Please make a vault action isolate. so in main we just need to call example `vault.LoadEnv()` or etch

regarding to the load technic. is should be loaded at beginning, we don't want reload. we should make sure we got the config ASAP